src/dune_engine/dune_project.ml

```diff
@@ -908,6 +915,11 @@ let parse ~dir ~lang ~file ~dir_status =
           let root = dir in
           let file_key = File_key.make ~name ~root in
           let dialects =
+            let dialects =
+              match Univ_map.find extension_args melange_extension with
+              | Some () -> (Loc.none, Dialect.rescript) :: dialects
```

Comment:
Do we not have the loc of where the extension is declared?

Reply:
I'm not sure how to get it. 

It's stored in `Extension.instance` but `Univ_map.find` returns `unit`